kong/clustering/rpc/manager.lua

```diff
@@ -244,18 +250,18 @@ function _M:_meta_call(c, meta_cap, node_id)
   local payload = cjson_decode(data)
   assert(payload.jsonrpc == "2.0")
 
+  -- now we only support snappy
+  if payload.result.rpc_frame_encoding ~= RPC_SNAPPY_FRAMED then
+    return nil, "unknown encoding: " .. payload.result.rpc_frame_encoding
+  end
+
```

Comment:
<img width="460" alt="image" src="https://github.com/user-attachments/assets/6b931321-ceb2-42de-bad8-f545b84f4934">

I found a typo in front of this modified lines, we could fix in this pr also.

`return nil, "wrong frame type: " .. type` should be `return nil, "wrong frame type: " .. typ`

Its a typo.

Reply:
done, thanks.